packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -268,6 +268,20 @@ export class PluginViewRegistry implements FrontendApplicationContribution {
                 }
             }
         }));
+
+        const focusCommandId = `${view.id}.focus`;
+        toDispose.push(this.commands.registerCommand({
+            id: focusCommandId,
+            label: 'Focus ' + view.name + ' View'
```

Comment:
Not intensional... Just followed the pattern for `toggleCommandId` in lines 208-211 of the same file.

Reply:
But actually specifying a property, like `label: 'Focus ${view.name} View'` doesn't work. It gives:
![Screenshot from 2020-02-07 16-13-51](https://user-images.githubusercontent.com/620781/74041137-63cd3800-49c5-11ea-85df-3f7f4cdc0921.png)
instead of:
![Screenshot from 2020-02-07 16-16-43](https://user-images.githubusercontent.com/620781/74041203-82cbca00-49c5-11ea-8a0f-dcfd8d63e972.png)
So, probably the only way to make a label here is the concatenation.

Reply:
did you use backtick ` ?

Reply:
Yes. I can make it to be like focusCommandId...

Reply:
> did you use backtick ` ?
No, I didn't. That's why it didn't work for me. Now it's fixed (I'm preparing the fixup)